### PR TITLE
fix(core): refactor webgl positioning to account for webglcontext viewframe

### PR DESCRIPTION
**Pull request by jmbuss**
## Overview
Update webgl positioning functions to make use of the viewframe passed to the webglcontext. 

## Tests
[Include a link to the passing GitHub action running the test suite here.]

## Legal
This project is available under the [Apache 2.0 License](http://www.apache.org/licenses/LICENSE-2.0.html).

### CODE REVIEW

**Review by square-li — APPROVED** [review]

File: packages/iot-app-kit-visualizations/src/components/common/webGLPositioning.ts [hunk 1]
```diff
@@ -13,18 +14,22 @@ export interface ClipSpaceRect {
  * This will return us the coordinates of a rectangle within clip space coordinates (i.e. coordinate space for webGL)
  * https://developer.mozilla.org/en-US/docs/Web/API/WebGL_API/WebGL_model_view_projection to learn more about clip space
  */
-const clipSpaceRect = (containerRect: RectScrollFixed, canvasRect: DOMRectReadOnly): ClipSpaceRect => {
+const clipSpaceRect = (
+  containerRect: RectScrollFixed,
+  canvasRect: DOMRectReadOnly,
+  { x: viewFrameX, y: viewFrameY }: ReturnType<typeof scrollPosition>
```
Comment (tracy-french) [inline packages/iot-app-kit-visualizations/src/components/common/webGLPositioning.ts:20]: Nice!

**Review by tracy-french — APPROVED** [review]
Awesome! Thank you for the fix @jmbuss!

File: packages/iot-app-kit-visualizations/src/components/sc-size-provider/sc-size-provider.tsx [hunk 2]
```diff
@@ -66,7 +65,7 @@ export class ScSizeProvider {
 
   setRect = () => {
     if (this.el && this.el.isConnected) {
-      const newRect = rectScrollFixed(this.el);
+      const newRect = webGLRenderer.getRectScrollFixed(this.el);
```
Comment (chejimmy) [inline packages/iot-app-kit-visualizations/src/components/sc-size-provider/sc-size-provider.tsx:68]: 🧑‍🔧 fixed

**Review by chejimmy — APPROVED** [review]

**Comment by boweihan** [comment]
Awesome!!

**Review by corteggiano — APPROVED** [review]

**Pull request merged**